### ISSUE TITLE
Possible example?

### ISSUE THREAD

**Issue by karllandheer** [lab-midas/med_segmentation#21]
Would it be possible to publish an example case such that we could see how to run this pipeline practically? For example just a folder containing a 3D data set, where when you run the pipeline the segmentation maps are produced? Would help a lot with user adoption.

**Comment by thomaskuestner** [comment]
I agree that this would help, but for now we don't have any datasets that we could share. We will think about generating a volunteer or phantom dataset.
